extensions/amp-subscriptions/0.1/test/test-local-subscriptions.js

```diff
@@ -92,6 +92,8 @@ describes.fakeWin('LocalSubscriptionsPlatform', {amp: true}, env => {
   it('should fetch the entitlements on getEntitlements', () => {
     const fetchStub = sandbox.stub(localSubscriptionPlatform.xhr_, 'fetchJson')
         .callsFake(() => Promise.resolve({json: () => Promise.resolve(json)}));
+    sandbox.stub(localSubscriptionPlatform.serviceAdapter_,
```

Comment:
`serviceAdapter` variable is available in this context. No need to reach for a private var.

Reply:
done